### PR TITLE
WFO module

### PR DESCRIPTION
**Pull request by nicefish-64bit**

### CODE REVIEW

**Comment by itsrachelfish** [comment]
Code looks good, but I'm not a fan of that indenting style!

**Comment by nicefish-64bit** [comment]
Oops forgot to commit that one.

File: modules/wfo.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+    });
+
+    if (this.current_state == this.STATE_NORMAL) {
+        // feed gives newest entries first, but we should
+        // announce oldest first.
+        need_announce.reverse();
+
+        if (need_announce.length >= this.LIMIT) {
+
+            announce_fn(util.format("9[WFO] At least9 %d posts were made in the last %d minutes! Join the action at 9https://wetfishonline.com/",
```
Comment (itsrachelfish) [inline modules/wfo.js]: I think it should only say "at least" if there were more changes than can be seen in a single page of RSS results. By default it should say specifically how many posts there are.

File: modules/wfo.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        });
+        feedparser.on('finish', function () {
+            var stream = this;
+
+            // We get here even when there's an error.
+            if (wfo.buffer.length > 0) {
+                wfo.state.newItems(wfo.buffer,
+                                   function (line) {
+                                       wfo.client.say('#botspam', line);
+                                   });
```
Comment (itsrachelfish) [inline modules/wfo.js]: I think it's kind of awkward to pass ```announce_fn``` as an argument like this

**Pull request merged**